fundamentals/apple/gettingstarted/final/Favourites/Shared/FavouritesApp.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  FavouritesApp.swift
+//  Shared
+//
+//  Created by Peter Friese on 05.07.22.
+//  Copyright © 2021 Google LLC. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import Firebase
+import FirebaseCore
+
+class AppDelegate: NSObject, UIApplicationDelegate {
+  func application(_ application: UIApplication,
+                   didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
+    FirebaseApp.configure()
+    Firebase.Analytics.setAnalyticsCollectionEnabled(true)
```

Comment:
nit: specifying the module here should be unnecessary, and also the top-level Firebase module is deprecated. If you need to specify the module, it should be `FirebaseAnalytics.Analytics`.